client/src/App.js

```diff
@@ -22,7 +23,13 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/movielist" component={MovieList} />
```

Comment:
repeat of my comment from your last branch, but this path should be "/" so the user sees it as soon as the app spins up